lib/pipeline/addon/components/settings-git/template.hbs

```diff
@@ -57,6 +57,62 @@
       </div>
     </section>
 
+    <section class="box mt-30">
+      <h3>{{t 'authPage.setting.limit.label'}}</h3>
```

Comment:
We're mixing single and double quotes in the same html file. We dont have a template linter yet (coming in the next ember upgrade) but when we do the standard is double qoutes in HBS files.

If you are working on an HBS file I am okay if you do a global (file global) find and replace of single quotes to double quotes.